src/Twig/TwigExtension.php

```diff
@@ -132,7 +135,7 @@ public function formatPercent($value): string
 
     private function pathPrefix(): string
     {
-        if ($this->pathPrefix !== null) {
+        if ($this->pathPrefix !== '') {
```

Comment:
no no no. this is breaking change.

null means use autodetection, having string value means use value from config.